app/models/agendapunt.js

```diff
@@ -1,22 +1,20 @@
-import Model from 'ember-data/model';
-import attr from 'ember-data/attr';
-import { belongsTo, hasMany } from 'ember-data/relationships';
+import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
 
-export default Model.extend({
-  uri: attr(),
-  beschrijving: attr(),
-  geplandOpenbaar: attr(),
-  heeftOntwerpbesluit: attr(),
-  titel: attr(),
-  type: attr('string-set'),
-  position: attr('number'),
-  vorigeAgendapunt: belongsTo('agendapunt', { inverse: null }),
-  referenties: hasMany('agendapunt', { inverse: null }),
-  zitting: belongsTo('zitting'),
-  behandeling: belongsTo('behandeling-van-agendapunt'),
-  publications: hasMany('published-resource', { inverse: null }),
+export default class AgendapuntModel extends Model {
+  @attr uri;
+  @attr beschrijving;
+  @attr geplandOpenbaar;
+  @attr heeftOntwerpbesluit;
+  @attr titel;
+  @attr('string-set') type;
+  @attr('number') position;
```

Comment:
The number transform can also be dropped [since the API already sends it as one](https://github.com/lblod/app-gn-publicatie/blob/7e5cac851b0007f7d65c1b90045ea80360c4e6fa/config/resources/slave-besluit-domain.lisp#L10).